app/metrics.yaml

```diff
@@ -114,22 +114,22 @@ events:
         description: "The preference key for the boolean (true/false) preference the user toggled. We currently track:
           show_search_suggestions, remote_debugging, telemetry, tracking_protection, search_bookmarks,
           search_browsing_history, show_clipboard_suggestions, show_search_shortcuts, open_links_in_a_private_tab,
-          pref_key_sync_logins, pref_key_sync_bookmarks and pref_key_sync_history"
+          pref_key_sync_logins and pref_key_show_search_suggestions_in_private"
       enabled:
         description: "Whether or not the preference is *now* enabled"
     bugs:
     - https://github.com/mozilla-mobile/fenix/issues/975
     - https://github.com/mozilla-mobile/fenix/issues/5094
     - https://github.com/mozilla-mobile/fenix/issues/5737
     - https://github.com/mozilla-mobile/fenix/issues/5586
-    - https://github.com/mozilla-mobile/fenix/issues/6396
```

Comment:
Did you intend to rename this? Maybe this was a bad merge? :)

Reply:
Yes, there was a strange merge...